Base/Models/Public/Include/Models/Models.hpp

```diff
@@ -0,0 +1,142 @@
+#pragma once
+
+#include "UnifiedModel.hpp"
+
+#include <map>
+#include <utility>
+
+namespace Models::New
```

Comment:
Try avoid New word. You may use V2 or ORM

Reply:
Done